deployments/engine/templates/deployment.yaml

```diff
@@ -44,6 +49,12 @@ spec:
             secretKeyRef:
               name: {{ .name }}
               key: {{ .secretAccessKeyKey }}
+        - name: PATH
```

Comment:
Again I'm not sure if we need this.  Have you tried without this?

Reply:
Let me investigate.

Reply:
- I don't think we need to set `$PATH` since we don't have the directory inside the container.
- Verified that `LD_LIBRARY_PATH`, etc. are set by the container runtime.

Will send a PR.